packages/react-query/src/FormatBalance.tsx

```diff
@@ -84,17 +86,17 @@ function FormatBalance ({ children, className = '', format, formatIndex, isShort
   // labelPost here looks messy, however we ensure we have one less text node
   return (
     <div className={`ui--FormatBalance ${className}`}>
-      {label ? <>{label}&nbsp;</> : ''}
-      <span
-        className='ui--FormatBalance-value'
-        data-testid='balance-summary'>{
+      {label ? <>{label}&nbsp;</> : ''}<span className='ui--FormatBalance-value'
+        data-testid='balance-summary' >{
           valueFormatted
             ? splitFormat(valueFormatted, labelPost, isShort)
             : value
               ? value === 'all'
                 ? t<string>('everything{{labelPost}}', { replace: { labelPost } })
                 : applyFormat(value, formatInfo, withCurrency, withSi, isShort, labelPost)
-              : `-${labelPost || ''}`
+              : labelPost instanceof String
```

Comment:
Is capitalisation of String here intentional? I think it should never be the case that `labelPost` is instance of `String` because String is not primitive type. See this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof#using_instanceof_with_string
This is why we had to cast `labelPost` to string in the line below, when normally the cast would not be required after the type check.

Could we do `typeof(labelPost) === string ? "-" + labelPost : labelPost` here instead?

Reply:
interesting, let me check....

Reply:
TSC is not happy with what you provided:
```
TS2693: 'string' only refers to a type, but is being used as a value here.
```
also linter: `ESLint: Operands of '+' operation must either be both strings or both numbers. Consider using a template literal. (@typescript-eslint/restrict-plus-operands)`

Reply:
But definitely sth is around here. It didn't look beauty after I introduced it and yes, I hoped someone would rant here :) Let's leave this discussion open.

Reply:
That warning is interesting. Have you quoted the word string?